README.md

```diff
@@ -1,2 +1,37 @@
-# intuit-login-page-clone
-This is a clone of intuit website's login page clone
+# intuit-signup-page-clone
+This is a clone of intuit website's signup page clone
```

Comment:
For users that don't know this website or want to learn more about the project, why not add the assignment link, the live link and the link of the webpage that is been a clone.

You could also add a brief description of **what, how and why** you worked on this project.

Reply:
Thanks for the comment. I'll add this.